setup.py

```diff
@@ -149,6 +149,7 @@ def clean_line(line):
     ],
     packages=find_packages(),
     install_requires=[
+        "hitran-api",
```

Comment:
I think having it under https://github.com/radis/radis/blob/develop/environment.yml is enough ?

```suggestion
```

Reply:
Removed from `environment.yml` as it's not a conda package.